Core/HLE/HLE.cpp

```diff
@@ -626,7 +626,7 @@ size_t hleFormatLogArgs(char *message, size_t sz, const char *argmask) {
 			break;
 
 		case 'i':
-			APPEND_FMT("%d", regval);
+			APPEND_FMT("%u", regval);
```

Comment:
:O okis thx the explication.
a little question, in Core/MIPS/MIPSVFPUUtils.cpp line 152:
```
void ReadVector(float *rd, VectorSize size, int reg) {
```
affects if we use u32 instead float to rd?
i think remove this type of lines can help:
```u32 *rdu = (u32 *)rd;```
convert float to u32 don't is a portable code...